source/content/guides/decoupled-sites/03-starter-kits.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Pantheon Decoupled
+subtitle: Starter Kits
+description: Use a Pantheon starter kit to congfire a decoupled site backend.
+categories: [platform]
+tags: [webops, workflow, decoupled]
+contributors: [joa-pan]
+type: guide
+layout: guide
+showtoc: true
+anchorid: starter-kits
+permalink: docs/guides/decoupled-sites/starter-kits/
+editpath: decoupled-sites/03-starter-kits.md
+reviewed: "2022-07-31"
+---
+
```

Comment:
Importing your GitHub repository into the Pantheon Decoupled app with the Node JS application. A change triggers a build -  will watch for changes in the main default branch - this is for the previewed site, will soon be available for Multidevs ( PRs against the default branch)